src/controllers/beneficial.owner.individual.controller.ts

```diff
@@ -1,160 +1,25 @@
 import { NextFunction, Request, Response } from "express";
-import { Session } from "@companieshouse/node-session-handler";
-
-import {
-  getFromApplicationData,
-  mapDataObjectToFields,
-  mapFieldsToDataObject,
-  prepareData,
-  removeFromApplicationData,
-  setApplicationData
-} from "../utils/application.data";
-import { saveAndContinue } from "../utils/save.and.continue";
-import { ApplicationDataType } from "../model";
-import { logger } from "../utils/logger";
-import {
-  BeneficialOwnerIndividualKey,
-  BeneficialOwnerIndividualKeys
-} from "../model/beneficial.owner.individual.model";
-import {
-  AddressKeys,
-  BeneficialOwnerNoc,
-  HasSameResidentialAddressKey,
-  ID,
-  InputDateKeys,
-  IsOnSanctionsListKey,
-  NonLegalFirmNoc,
-  TrusteesNoc,
-} from "../model/data.types.model";
-import {
-  ServiceAddressKey,
-  ServiceAddressKeys,
-  UsualResidentialAddressKey,
-  UsualResidentialAddressKeys,
-} from "../model/address.model";
-import {
-  DateOfBirthKey,
-  DateOfBirthKeys,
-  StartDateKey,
-  StartDateKeys,
-} from "../model/date.model";
-import { BENEFICIAL_OWNER_INDIVIDUAL_PAGE, BENEFICIAL_OWNER_TYPE_URL } from "../config";
-import { v4 as uuidv4 } from "uuid";
 
+import { getBeneficialOwnerIndividual, getBeneficialOwnerIndividualById, postBeneficialOwnerIndividual, removeBeneficialOwnerIndividual, updateBeneficialOwnerIndividual } from "../utils/beneficial.owner.individual";
```

Comment:
maybe have one imported name per line.

Reply:
good idea, was considering this originally